content/guides/component-testing/custom-mount-vue.md

```diff
@@ -270,6 +247,63 @@ components that use all of Vue's language features.
 
 Congrats! Happy building. 🎉
 
+### Vuetify
+
+<code-group>
+<code-block label="Vuetify 3" active>
+
+```js
+import { mount } from 'cypress/vue'
+import { createVuetify } from 'vuetify'
+import { mergeDeep } from 'vuetify/lib/util/helpers'
+import 'vuetify/styles'
+
+Cypress.Commands.add('mount', (component, options, vuetifyOptions) => {
+  const vuetify = createVuetify(vuetifyOptions)
+  const defaultOptions = {
+    global: {
```

Comment:
I wonder if we should just make a third party library to handle all this complexity, eg `cypress-vuetify`, or something like that. The fact the docs are so complex indicates there might be value in abstracting this 🤔